WebRoot/js/zimbraAdmin/globalconfig/controller/ZaGlobalConfigViewController.js

```diff
@@ -419,6 +486,26 @@ function () {
 	return true;
 }
 
+ZaGlobalConfigViewController.isValidHostName =
+function (isHostNameStr) {
+	if (!isHostNameStr) {
```

Comment:
minor: indentation

Reply:
I see line of code "if (!isHostNameStr) {"  is indented. Please review again or tell me what need to be changed at that line!